backend/tests/Feature/Web/Admin/MaterialTypeOptionalTest.php

```
@@ -0,0 +1,92 @@
+            ])
+            ->assertRedirect(route('admin.materials.index'));
+
+        $this->assertDatabaseHas('materials', [
+            'code' => 'NO-TYPE-1',
+            'material_type_id' => null,
+        ]);
+    }
+
+    public function test_material_can_still_be_created_with_a_material_type(): void
+    {
+        $type = MaterialType::create(['code' => 'RAW', 'name' => 'Raw']);
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\MaterialType::create().


(staticMethod.notFound)